WordPress/Classes/ViewRelated/Domains/Views/DomainResultView.swift

```diff
@@ -0,0 +1,114 @@
+import SwiftUI
+
+struct DomainResultView: View {
+
+    @State private var subtitleHeight: CGFloat = .zero
+    let domain: String
+    let dismiss: () -> Void
+
+    private struct AttributedLabel: UIViewRepresentable {
+        typealias UIViewType = UILabel
+        @Binding var dynamicHeight: CGFloat
+
+        func makeUIView(context: Context) -> UILabel {
+            let label = UIViewType()
+            label.numberOfLines = 0
+            label.setContentCompressionResistancePriority(.defaultLow, for: .horizontal)
+            label.lineBreakMode = .byWordWrapping
+            return label
+        }
+
+        func updateUIView(_ uiView: UILabel, context: Context) {
+            configuration(uiView)
+
+            // Makes the label vertically hug its multi-line content
+            DispatchQueue.main.async {
+                dynamicHeight = uiView.sizeThatFits(CGSize(width: uiView.bounds.width, height: CGFloat.greatestFiniteMagnitude)).height
+            }
+        }
+
+        var configuration: (UIViewType) -> Void
+    }
+
+    var body: some View {
+        ZStack {
+            VStack(alignment: .leading, spacing: Metrics.interItemVerticalPadding) {
+
+                Image("Domains Lockup")
+                    .padding(Metrics.logoPadding)
+                    .accessibility(hidden: true)
+
+                Text(TextContent.title)
+                    .multilineTextAlignment(.leading)
+                    .font(.largeTitle.bold())
+                    .foregroundColor(Color.primary)
+
+                AttributedLabel(dynamicHeight: $subtitleHeight) {
+                    $0.attributedText = subtitleWith(domain: domain)
+                }.foregroundColor(Color(UIColor.muriel(color: .text)))
```

Comment:
Code formatting could be improved by aligning all attributes:

```
HStack {
}
.attribute()
.attribute()
```

instead of:
```
HStack {
}.attribute()
.attribute()
```

Reply:
Good call! Fixed